### DISCUSSION TITLE
sudo gibt segfault

### DISCUSSION THREAD

**Discussion by pony1k** [Freetz-NG/freetz-ng#341] [Allgemein]
Hallo, ich habe FritzOS 07.27 rev88562 mit Freetz ng-18431M-b6cb077eb.
Ich möchte sudo benutzen, bekomme aber immer
```
doctor@tardis:~$ sudo ls
Segmentation fault
```
Woran kann das bloß liegen?

**Comment by Conan179** [comment]
Build umgebung oder auf der box?

**Reply by pony1k** [reply]
Auf der Box.

**Comment by fda77** [comment]
`strace -fff sudo` könnte helfen
Paket vorher halt noch installieren
